public/imprint.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="favicon.ico" />
+    <link href="./legal.css" rel="stylesheet" type="text/css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Terms" />
+    <title>Kilt Stakeboard - Imprint</title>
+  </head>
+
+  <body>
+    <main class="container">
+      <div class="content">
+        <h1 class="heading">Imprint</h1>
+        <section class="section">
+          <p class="imprintLine">
+            B.T.E. BOTLabs Trusted Entity GmbH
+          </p>
+          <p class="imprintLine">Keithstraße 2-4</p>
+          <p class="imprintLine">10787 Berlin, Germany</p>
+        </section>
+
+        <section>
+          <p class="imprintLine">German Commercial Court:</p>
+          <p class="imprintLine">
+            Amtsgericht Charlottenburg in Berlin
+          </p>
+          <p class="imprintLine">Registration Number: HRB 231219B</p>
+          <p class="imprintLine">VAT No: DE 346528612</p>
+          <p class="imprintLine">Managing Director: Ingo Rübe</p>
+        </section>
+
+        <section>
+          <p class="imprintLine">
+            Contact:
+            <a class=anchor href="mailto:info@botlabs.org">
+              info@botlabs.org
+            </a>
+          </p>
+          <p class="imprintLine">
+            Or go to
+            <a
+              class="anchor"
+              href="https://support.kilt.io/support/home"
+              target="_blank"
+              rel="noreferrer">
+              Tech support
+            </a>
+            and click on "Contact Us"
+          </p>
+        </section>
+
+        <p class="section">
+          Requirements according to § 5 TMG (Germany)
+        </p>
+      </div>
+    </main>
+
+    <nav class="footerContainer">
```

Comment:
I think it would be better semantically to use the `<footer>` tag for the footer, and have the nav menu inside it.

This also applies to the other pages.